sphinx/builders/gettext.py

```diff
@@ -216,7 +216,8 @@ def finish(self):
                     if self.config.gettext_location:
                         # generate "#: file1:line1\n#: file2:line2 ..."
                         pofile.write("#: %s\n" % "\n#: ".join(
-                            "%s:%s" % (safe_relpath(source, self.outdir), line)
+                            "%s:%s" % (safe_relpath(source, self.outdir).
+                                       replace('\\', '/'), line)
```

Comment:
Could you use `os.sep` and `sphinx.util.SEP` for this?  (Maybe, adding an inverse function for `sphinx.util.osutil.os_path` is appropriate?)